src/Uno.UI.RuntimeTests/Tests/Microsoft_UI_Xaml_Controls/Given_PipsPager.cs

```diff
@@ -0,0 +1,50 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Shapes;
+using Microsoft.UI.Xaml.Controls;
+using Private.Infrastructure;
+using Uno.UI.RuntimeTests.Helpers;
+
+#if !HAS_UNO_WINUI
+using Windows.UI.Xaml.Controls;
+#endif
+
+namespace Uno.UI.RuntimeTests.Tests.Microsoft_UI_Xaml_Controls;
+
+[TestClass]
+public partial class Given_PipsPager
+{
+	// <PipsPager>
+	// <PipsPager.NormalPipStyle>
+	// <Style TargetType="Ellipse">
+	// <Setter Property="Fill" Value="Red"/>
+	// </Style>
+	// </PipsPager.NormalPipStyle>
+	// </PipsPager>
+
+	[TestMethod]
+	[RunsOnUIThread]
+	public async Task When_MaxVisiblePips_GreaterThan_NumberOfPages()
+	{
+		var SUT = new PipsPager
+		{
+			NumberOfPages = 7,
+			MaxVisiblePips = 5
+		};
+
+		await UITestHelper.Load(SUT);
+
+		var initialScreenshot = await UITestHelper.ScreenShot(SUT);
```

Comment:
You may need to exclude some platforms that don't support screenshots by using ApiInformation (unless it's already done in UITestHelper.ScreenShot).